community/kernel/pom.xml

```diff
@@ -134,7 +134,7 @@ the relevant Commercial Agreement.
         <configuration>
           <dotGitDirectory>${git.directory}</dotGitDirectory>
           <failOnNoGitDirectory>false</failOnNoGitDirectory>
-          <useNativeGit>true</useNativeGit>
+          <useNativeGit>false</useNativeGit>
```

Comment:
Right, it sounds reasonable. Can we please have this change in its own PR?
This sounds completely unrelated to core-edge.